File: leap_year.py
def year(n):
    if n%4==0 and n%100!=0 or n%400==0:
        return True
    else:
        return False
print(year(2000))


#leap year to find the year would be leap year or not

# to modulus the year by 4 and 100 or 400 to get the result as 0 is a leap year otherwise not a leapyear

File: c++/leap-year.cpp
#include <iostream>
#include <iomanip>
using namespace std;
int main(){
  int year;
  cin>>year;
  if(year%4==0 && year%100!=0 || year%400==0){
    cout<<"leap year"<<endl;
  }
  else{
    cout<<"Not leap year"<<endl;
  }
}